packages/SwingSet/test/vat-admin/test-create-vat.js

```diff
@@ -258,8 +258,31 @@ function findRefs(kvStore, koid) {
   const refcount = refcountString.split(',').map(Number);
   return { refs, refcount };
 }
-
-test('createVat holds refcount', async t => {
+/**
+ * TODO Figure out why #5892 causes a refcount difference, and
+ * unskip this.
```

Comment:
@FUDCo @warner Please see scary refcount change. Thanks.

Reply:
(This test exercises the refcount maintained, by the kernel, on any objects in the `vatParameters` that are passed through a create-vat request. We call `E(vatAdminService).createVat(bundleID, { vatParameters: [ object, devicenode] })`, and look at the refcount on that exported `object`)

vat-vat-admin receives the following deliveries:
* `getBundleCap()`
* `bringOutYourDead` (1)
* `createVat()`
* `bringOutYourDead` (2)
* `newVatCallback()`
* `bringOutYourDead` (3)
* `dropExports`, then `retireExports`

The `object` reference is imported into vat-vat-admin as an argument of the `createVat()` call.

My first pass at analysis shows that the `object` reference was previously dropped during the (3) BOYD call, just after `newVatCallback`, whereas in this PR's branch, it is being dropped during the (2) BOYD call, just after `createVat`. This implies that vat-vat-admin was retaining a reference to the `object` Presence longer in the previous version.

I'm still investigating what might have changed here.

Reply:
I don't see any reason why vat-vat-admin would retain a reference to `vatParameters` after the initial `createVat()` delivery: they are passed into a device call during `createVat()` and then not retained anywhere. So it seems like the new behavior is the correct one, and the previous retention was the surprise. I'll keep digging.

Reply:
I found that changing `track-turns.js` to hoist the "add rejection node" closure to its own top-level function (thus removing `funcs` and `args` from its lexical scope) caused the same test-create-vat.js failure as did this PR (disabling track-turns entirely). Given how `trackTurns` is used by `handled-promise.js`, I believe that holding `funcs` will cause swingset to hold onto unserialized delivery arguments (i.e. Presences) longer than expected, and I think that's what was causing this test to change behavior.

I'm writing up an Endo ticket to address this problem. In the meantime, I think we should land this PR, and then fix the swingset test to expect the (now-understood-to-be-correct) ordering of events. I'll take care of that (and unskip this test case) after this PR lands.

Reply:
https://github.com/endojs/endo/issues/1245 is the Endo ticket on the object retention